modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/constants/SegmentsExperimentProductNavigationConstants.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.segments.experiment.web.internal.constants;
+
+/**
+ * @author Renan Vasconcelos
+ */
+public class SegmentsExperimentProductNavigationConstants {
+
```

Comment:
@RenanUmV, rename to `ProductNavigationControlMenuEntryConstants`, please.

PS: apply to the other constant classes as well. Thx